webroot/index.html

```diff
@@ -35,6 +35,8 @@
     <script src="//unpkg.com/showdown/dist/showdown.min.js" defer></script>
     <script type="module" src="https://cdn.jsdelivr.net/npm/@justinribeiro/lite-youtube@0.6.2/lite-youtube.js" defer></script>
 
+    <!-- moment js -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.0/moment.min.js" integrity="sha512-Izh34nqeeR7/nwthfeE0SI3c8uhFSnqxV0sI9TvTcXiFJkMd6fB644O64BRq2P/LA/+7eRvCw4GmLsXksyTHBg==" crossorigin="anonymous"></script>
```

Comment:
If we are going to use any library for this, let's please go with a lightweight one and not the behemoth library that is momentjs. An alternative would be something like https://github.com/iamkun/dayjs

Reply:
Oh nice find! @graywolf336

Though for this particular task it doesn't seem necessary to pull in another external dependency, since we're not doing another date math anywhere else in the app. But you can totally borrow code from it to see how they do date calculations and put it in a util file for the messages @ahmadkarlam!

Reply:
@ahmadkarlam have a look at the Javascript Date API, there are some pretty good functions in there you can use to do some calculations in probably the same amount of code you'd need with an external library.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

You can some calculations with just new Date() which gets now, and new Date(messageTimestamp), and do some math on the getHour() of their results.

Also, no need to worry about converting timezones or anything like that. Just using your computer's local date for this is good enough.  `toLocaleString()` and `toLocaleTimeString()` seem like a pretty good options.